Gemfile.lock

```diff
@@ -652,10 +654,10 @@ GEM
       xmlmapper (~> 0.6)
     xmlmapper (0.7.2)
       nokogiri (~> 1.5)
-    xpath (2.0.0)
+    xpath (2.1.0)
       nokogiri (~> 1.3)
     zonebie (0.6.1)
-    zxcvbn-js (4.4.2)
+    zxcvbn-js (4.4.3)
```

Comment:
Looks like the Ruby gem's versioniong does not necessarily correspond to the Dropbox versioning. It looks like he made a change to the Ruby code and updated the version number, so I think we're good to go here.
https://github.com/bitzesty/zxcvbn-js/commits/master